adagrams/game.py

```diff
@@ -1,11 +1,156 @@
+from operator import le
```

Comment:
It looks like `le`, which is part of this `import` statement never got used. Most likely this is because of a "helpful" VSCode feature that will attempt to add `import` statements for you automatically, but often just cause problems. I would recommend just turning it off. Ansel provided these steps:

1. Open the Settings screen, by either going to the Code > Preferences > Settings menu pick, or using the ⌘, keyboard shortcut.
2. In the Search settings, look for: analysis.
3. Look for a result titled Python > Analysis: Auto Import Completions.
4. Uncheck the option to Offer auto-import completions.